Assets/Scripts/SS3D/Systems/Health/BodyParts/Bodypart.cs

```diff
@@ -259,12 +248,11 @@ protected virtual void CopyValuesToBodyPart(BodyPart bodyPart)
     /// All child body parts are detached, all internal body parts are destroyed.
     /// </summary>
     [Server]
-    private void DestroyBodyPart()
+    protected void DestroyBodyPart()
```

Comment:
Same for this one, keep it private, use events or abstract methods to implement particular behavior.